### ISSUE TITLE
Bug runing videplayer in IOS

### ISSUE THREAD

**Issue by maxTeste** [innovationmauritiustelecom/video_player_360#20]
hi, 
i'm trying to run video player plugin in IOS and when i call the function: VideoPlayer360.playVideoURL, it's bugged like the sensor not working correctly.
Anyone that run this plugin in Ios and is working correctly?

**Comment by stephangopaul** [comment]
@rubenteste, yes there is an issue with iOS 14.4 which is causing the sensors to keep spinning.
You need to restart your device to temporarily fix this.
Since the Google VR SDK for iOS is deprecated, we will need to replace it with another VR SDK.
 Unfortunately we do not have an ETA for this. :(

**Comment by stephangopaul** [comment]
Reopened for reference.
